pom.xml

```diff
@@ -20,8 +20,8 @@
     <jackson.version>2.2.3</jackson.version>
     <jenkins.version>2.7.1</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
-    <node.version>5.8.0</node.version>
-    <npm.version>3.7.3</npm.version>
+    <node.version>6.4.0</node.version>
+    <npm.version>3.10.3</npm.version>
```

Comment:
No, we have maven profiles that use the maven download plugin to download the node and npm tars (the frontend plugin just executes node/npm commands, assuming they are already in place). The slave env has a particular arch that these profiles look at before deciding which tars to download for that arch. That part was failing for the newer versions, but was fine when I dropped back to these versions (which are also "fairly new"). We can try again but I think these are fine for now.